.github/actions/setup/action.yml

```diff
@@ -46,3 +46,7 @@ runs:
     # - if: steps.nx-cache.outputs.cache-hit != 'true'
     - run: npm run build
       shell: bash
+    - name: Run step only if DEPLOYMENT_ENVIRONMENT is CDN
```

Comment:
With this env variable, references to @coveo/headless and @coveo/atomic (and bueno once added to the CDN) will be replaced by the corresponding CDN url